src/ekss/config.py

```diff
@@ -48,6 +41,12 @@ class VaultConfig(BaseSettings):
         example="example_secret",
         description="Vault secret ID to access a specific prefix",
     )
+    ca_bundle_location: Optional[Path] = Field(
+        None,
+        example="/my_bundle.pem",
+        description="Path pointing to the location of the CA bundle to use when"
+        + " communicating with the vault.",
```

Comment:
```suggestion
    vault_verify: Union[bool, str] = Field(
        True,
        example="/etc/ssl/certs/my_bundle.pem",
        description="SSL certificates (CA bundle) used to"
          " verify the identity of the vault, or True to"
          " use the default CAs, or False for no verification.",
```

Rationale:
- makes the setting more descriptive (`ca_bundle_location` does not say for what the CA is used)
- if you know requests, you already know what this parameter means
- httpx and requests expect `str` instead of `Path`
- the two boolean settings can also sometimes be useful